docs/usage/handheld/miyoocfw.md

```diff
@@ -0,0 +1,63 @@
+# MiyooCFW
+
+[MiyooCFW](https://github.com/TriForceX/MiyooCFW/wiki) ([Code](https://github.com/TriForceX/MiyooCFW)) is a custom firmware  for BittBoy, PocketGo, PowKiddy V90-Q90-Q20 and third party handheld consoles. It is based on the buildroot build environment and loosely based on OpenDingux. While it is intended for the named less powerful handhelds, it packs a good and wide selection of emulators. Some tinkering with the BIOS files is required though, but no worries, most of that is covered below or in their [Wiki](https://github.com/TriForceX/MiyooCFW/wiki/Emulator-Info)
+
+## BIOS Files
+
+MiyooCFW doesn't seem to have a centralized folder for putting BIOS files so it's a chore to prepare the card for running the provided emulators. Here's a try to help with finding the right ones and putting them in the right place.
+
+| System   | Emulator | Folder | Filenames | MD5SUMs | Comments |
+|----------|----------|--------|-----------|---------|----------|
+|  GB/GBC  | Gambatte | /.gambatte/bios/BIOS | `gb_bios.bin` &#10; `gbc_bios.bin` |  `32fbbd84168d3482956eb3c5051637f5` &#10; `dbfce9db9deaa2567f6a84fde55f9680`      |  Only required for authentic boot screen         |
```

Comment:
omg I didn't know you could `&#10;` with Markdown tables, I've definitely wanted to use it in the past!

Reply:
> omg I didn't know you could `&#10;` with Markdown tables, I've definitely wanted to use it in the past!

It turns out it does not in fact work :/. Markdown lint doesn't want me to use <br /> but if you are ok with it, I'd use it anyway

Reply:
Looks like the linter is passing 🤷 do you know which rule number it was failing?